TYPO3.Media/Classes/TYPO3/Media/Validator/ImageTypeValidator.php

```diff
@@ -21,7 +21,7 @@
  * Validator that checks the type of a given image
  *
  * Example:
- * [at]Flow\Validate("$image", type="\TYPO3\Media\Validator\ImageTypeValidator", options={ "allowedTypes"={"jpeg", "png"} })
+ * [at]Flow\Validate("$image", type="\Neos\Media\Validator\ImageTypeValidator", options={ "allowedTypes"={"jpeg", "png"} })
```

Comment:
Even though this is just a comment, it will lead people to the use of a not (yet) existing class. So, please only merge with the actual vendor name change.